src/OmnichannelChatSDK.ts

```diff
@@ -1619,6 +1619,41 @@ class OmnichannelChatSDK {
         }
     }
 
+    public async sendReadReceipt(messageId: string): Promise<void> {
+        if (!this.isInitialized) {
+            this.debug && console.warn('[OmnichannelChatSDK][sendReadReceipt] SDK not initialized');
+            return;
+        }
+
+        if (this.liveChatVersion !== LiveChatVersion.V2) {
+            this.debug && console.warn('[OmnichannelChatSDK][sendReadReceipt] Read receipts only supported for LiveChat V2');
+            return;
+        }
+
+        if (!messageId) {
+            this.debug && console.warn('[OmnichannelChatSDK][sendReadReceipt] Message ID is required');
+            return;
+        }
+
+        try {
+            await (this.conversation as ACSConversation).sendReadReceipt(messageId);
+
+            this.scenarioMarker.singleRecord(TelemetryEvent.SendReadReceiptSucceeded, {
```

Comment:
review the use of the scenario pattern for telemetry, this is not how is in use in other methods